rcl/test/rcl/test_info_by_topic.cpp

```diff
@@ -295,11 +299,11 @@ TEST_F(CLASSNAME(TestInfoByTopicFixture, RMW_IMPLEMENTATION),
 TEST_F(CLASSNAME(TestInfoByTopicFixture, RMW_IMPLEMENTATION),
   test_rcl_get_publishers_subscription_info_by_topic)
 {
-  // This is implemented only in fastrtps currently.
+  // This is implemented only in fastrtps and Cyclone DDS currently.
```

Comment:
nit: I think you can move this comment to https://github.com/ros2/rcl/pull/572.

Reply:
fixed